pkg/util/errutil/errors.go

```diff
@@ -2,23 +2,18 @@ package errutil
 
 import (
 	"fmt"
-
-	"golang.org/x/xerrors"
 )
 
-// Wrap is a simple wrapper around Errorf that is doing error wrapping. You can read how that works in
-// https://godoc.org/golang.org/x/xerrors#Errorf but its API is very implicit which is a reason for this wrapper.
-// There is also a discussion (https://github.com/golang/go/issues/29934) where many comments make arguments for such
-// wrapper so hopefully it will be added in the standard lib later.
+// Wrap is a simple wrapper around fmt.Errorf that wraps errors.
```

Comment:
I would suggest keeping the reference to the golang issue so if such a wrapper is introduced at some point we can remove this one.

Reply:
@papagian The wrapper is introduced actually, it's `fmt.Errorf("wrapping message: %w", err)`. I am open to replacing `errutil.Wrap*`, but I got the impression so far we're keeping it. Maybe we should schedule a team discussion wrt. potentially replacing it?

Reply:
ok, then I don't see why not removing it but I'm open to discuss it with the rest of the team.

Reply:
@papagian Shall we try to have a discussion when everyone is back from vacation then?

Reply:
It sounds good